src/Install/CodeEnvironment/OpenCode.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laravel\Boost\Install\CodeEnvironment;
+
+use Laravel\Boost\Contracts\Agent;
+use Laravel\Boost\Contracts\McpClient;
+use Laravel\Boost\Install\Enums\McpInstallationStrategy;
+use Laravel\Boost\Install\Enums\Platform;
+
+class OpenCode extends CodeEnvironment implements Agent, McpClient
+{
+    public function name(): string
+    {
+        return 'opencode';
+    }
+
+    public function displayName(): string
+    {
+        return 'OpenCode';
```

Comment:
I think they prefer either all lowercase
```suggestion
        return 'opencode';
```